setuptools/dist.py

```diff
@@ -314,12 +318,14 @@ def check_package_data(dist, attr, value):
                 iter(v)
             except TypeError:
                 break
+            if isinstance(v, str):
+                distutils.log.warn(
```

Comment:
I think we've generally been doing `warnings.warn` instead of `distutils.log.warn` these days.